package.json

```diff
@@ -20,21 +20,21 @@
   "author": "Jordan Foreman <me@jordanforeman.com>",
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.1126.0",
-    "yargs": "^15.0.0"
+    "aws-sdk": "^2.1469.0",
+    "yargs": "^17.7.2"
```

Comment:
:bulb: this is my only concern with this changeset. Need to make sure that two major version updates of a core dependency doesn't break the actual CLI. Appreciate you ensuring the tests pass, but have you also ran the CLI to ensure it is still able to pull a CloudFront alias?

related docs:

- [`yargs@v17`](https://github.com/yargs/yargs/releases/tag/v17.0.0)
- [`yargs@v16`](https://github.com/yargs/yargs/releases/tag/v16.0.0)

Reply:
Tested locally, and this works :+1: